scripts/build-bpf.sh

```diff
@@ -30,13 +30,14 @@ export V="${V:-1}"
 make clean 
 make  "${@:2}" 
 make cpyth 
+make cpythtest
 rm ./target/*-keypair.json
 
 
 #build Rust and link it with C
 cd "${PYTH_DIR}"
 cargo clean
-cargo test
+cargo test-bpf
```

Comment:
I'm not sure about the difference between both but solana encourages `cargo test-bpf`